app/models/work.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+
+class Work < ApplicationRecord
+  belongs_to :depositor,
+             class_name: 'User',
+             foreign_key: 'depositor_id',
+             inverse_of: 'works'
+
+  has_many :work_creations,
+           dependent: :restrict_with_exception
+
+  has_many :aliases,
```

Comment:
@rschenk seeing your other comment in the factory, it occurred to me that alias/aliases could pose problems. Do we want to rename this?

Reply:
It's only a problem in the factory, where `alias` could not be used as a bare word. I think we're good here and no need to rename